css/properties.json

```
@@ -4766,7 +4766,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/fill"
   },
   "fill-opacity": {
-    "syntax": "<alpha-value>",
+    "syntax": "<opacity>",
```

```suggestion
    "syntax": "<'opacity'>",
```

note: `<opacity>` is not equal to `<'opacity'>`

https://github.com/mdn/data/blob/8ec6d5057d8b774c185e33fe684b5af4dc8f3333/css/properties.json#L9576-L9591

---

Thanks!